docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,3931 @@
+* Adds the ability to make spaces plugin optional {kibana-pull}152115[#152115]
+* Adds proxy ssl key and certificate to agent policy {kibana-pull}152005[#152005]
+* Adds `_meta` field `has_experimental_data_stream_indexing_features` {kibana-pull}151853[#151853]
+* Adds the ability to create templates and pipelines when updating package of a single package policy from type integration to input {kibana-pull}150199[#150199]
+* Adds user's secondary authorization to Transforms {kibana-pull}154665[#154665]
+
+Infrastructure::
+Adds Memory Available Graph To Hosts View {kibana-pull}151863[#151863]
+
+Lens & Visualizations::
+* Adds the ability to sync the partition legend order with the filters order in *Lens* {kibana-pull}154820[#154820]
+* Adds support for icons in the new *Lens* metric {kibana-pull}154210[#154210]
+* Adds the ability to share with reports in *Lens* {kibana-pull}153429[#153429]
+* Adds show and hide heatmap ticks in *Lens* {kibana-pull}153425[#153425]
+* Adds the ability to remove empty headers when there is no x-axis in *Lens* {kibana-pull}153420[#153420]
+* Adds improvements to the Metric formatter to support bit format in *Lens* {kibana-pull}153389[#153389]
+* Adds the ability to prevent default behaviour from action callback in *Lens* {kibana-pull}152842[#152842]
+* Adds Random Sampling to *Lens* {kibana-pull}151749[#151749]
+
+Machine Learning::
+* DFA creation wizard: add ability to add time field to result data view {kibana-pull}155669[#155669]
+* Display info when no datafeed preview results are found {kibana-pull}155650[#155650]
+* Adding ignore unavailable indices option to anomaly detection job wizards {kibana-pull}155527[#155527]
+* Support multiple model deployments {kibana-pull}155375[#155375]
+* Uses two weeks before now for default start time in job start date picker {kibana-pull}155312[#155312]
+* AIOps: Adds filter action for the Change point detection results {kibana-pull}155256[#155256]
+* Adds search links for AIOps Labs pages {kibana-pull}155202[#155202]
+* AIOps: Adds field stats for metric and split fields {kibana-pull}155177[#155177]
+* AIOps: Link from Explain Log Rate Spikes to Log Pattern Analysis {kibana-pull}155121[#155121]
+* Explain Log Rate Spikes: adds popover to analysis table for viewing other field values {kibana-pull}154689[#154689]
+* Explain Log Rate Spikes: Makes use of random sampling for overall histogram chart {kibana-pull}154520[#154520]
+* Explain Log Rate Spikes: Adds table action to copy filter to clipboard {kibana-pull}154311[#154311]
+* Data Frame Analytics: support custom urls in jobs {kibana-pull}154287[#154287]
+* Change point detection: support for multiple metric and split fields {kibana-pull}154237[#154237]
+* Enhances support for counter fields in data visualizer / field statistics {kibana-pull}153893[#153893]
+* Explain Log Rate Spikes: Support to filter fields from grouping {kibana-pull}153864[#153864]
```

Comment:
This is already in the Features section. Is it OK to have it in here too?

Reply:
Fixed this one too.